f2/src/utils/formatAnalystEmail.js

```diff
@@ -18,13 +18,14 @@ const formatReportInfo = (data) => {
     selfHarmString = data.selfHarmWords
     returnString = `\n\n<h1>SELF HARM WORDS FOUND : ${selfHarmString}</h1>`
   }
+  const versionString = `${data.prodVersion} (${data.appVersion})`
```

Comment:
Is this correct?: In the email, I get '1.1.0 (no version)' What is the difference between prod and app versions? Should we make it clearer?

Reply:
👍 Would probably be useful to do something like
```
1.1.0 (GitHub: 2a324f)
```
locally and on Heroku the GitHub hash doesn't work - I'll make that more expanitory too.